packages/postgres-database/Makefile

```diff
@@ -67,6 +67,7 @@ setup-prod: guard-POSTGRES_DATA_VOLUME install-dev up-prod ## sets up a database
 
 .PHONY: setup-commit
 setup-commit: install-dev up-pg ## sets up a database to create a new commit into migration history
+	sleep 2
```

Comment:
?

Reply:
> ?

docker is starting too fast since one of the last upgrades. this sleep is not really critical!